apps/reference/docs/guides/api/generating-types.mdx

```diff
@@ -6,71 +6,62 @@ description: How to generate types for your API and Supabase libraries.
 
 Supabase APIs are generated from your database, which means that we can use database introspection to generate type-safe API definitions.
 
-### Generating types from OpenAPI specification
+### Generating types using Supabase CLI
 
-Supabase generates an OpenAPI specification file for your database which can be used to generate your data types for usage with TypeScript.
+The Supabase CLI is a single binary Go application that provides everything you need to setup a local development environment.
 
-The OpenAPI specification for your Supabase project can be accessed as follows:
+You can [install the CLI](https://www.npmjs.com/package/supabase) via npm or other supported package managers.
 
-```txt
-https://your-project.supabase.co/rest/v1/?apikey=your-anon-key
+```bash
+npm i supabase
```

Comment:
```suggestion
npm install --global supabase
```

Reply:
Local install is intentional. Keeps it versioned in package.json

Reply:
Then we should recommend `--save-dev`, otherwise it'll be installed as a dependency, which is almost never the right thing.

Reply:
Need to update cli readme as well